examples/federated_statistics/df_stats/df_stats_job/custom/tests/df_statistics_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pandas as pd
+
+from examples.federated_statistics.df_stats.df_stats_job.custom.df_statistics import DFStatistics
+from nvflare.apis.fl_context import FLContext
+
+
+class TestDFStatistics:
```

Comment:
This is just a regular unit test that has nothing specific to NVFlare though.

I am not sure if we should put this in the custom folder of a job.
These custom codes will be deployed to each client site.
I don't think every client site want these unit test codes?

Or what is the purpose of this code?